jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/VaultInventoryCollector.java

```diff
@@ -126,7 +126,7 @@ private void setSecretExpirationDate(Vault azureVault, VaultVH vaultVH) {
 		} catch (KeyVaultErrorException e) {
 			//if permission is denied to get list of secrets, then do not raise any violation
 			vaultVH.setSecretExpirationDate(new HashSet<>());
-			log.info(e.getMessage());
+			log.error(e.getMessage());
```

Comment:
@santhosh-challa I dont think this should be an error since we are handling a valid scenario. WE should log this as a simple log like along "skipping collecting asset type x. Permissions Denied" and create a record in Notifications with source = Paladin Cloud type = Permission